srcs/app/CGI/CGI.cpp

```diff
@@ -92,7 +89,12 @@ int CGI::ExecuteCGI(void) {
 			SyscallWrap::execveWr(exec_path_.c_str(), argv, CGIenv_,
 								  __FILE__, __FUNCTION__, __LINE__);
 		}
-		catch (const std::exception &) {
+		catch (const std::exception &e) {
```

Comment:
@gbudau : this is a bit hackish, but it works. Please, let me know what do you think about it.

Reply:
I took these changes into a different PR, since it's about an unrelated problem.